src/xcode/ENA/ENA/Source/HTTPClientCore/Service/Resource/_Resource.swift

```diff
@@ -14,12 +15,15 @@ protocol Resource {
 	var type: ServiceType { get }
 	var sendResource: Send { get }
 	var receiveResource: Receive { get }
-	
+
 	// Defines a default value for no network cases as the specific receive model (for resources like e.g. AppConfig, AllowList)
 	var defaultModel: Receive.ReceiveModel? { get }
-	
+	// Defines the trust evaluation for the certificate pinning.
 	var trustEvaluation: TrustEvaluating { get }
-	
+	// Indicates if the loading of the resource should be retried. Counts descending.
+	// NOTE: Compete logically with defaultModel: When setting both, we will first exhaust all retries and then when still failing we will return the defaultModel.
+	var retryingCount: Int? { get set }
```

Comment:
why make the count optional, zero as a default value might be good to indicate a no retrying?

Reply:
yeah, I will change this which comes with a change to thread-safety of this property.